sys-devel/llvm/patches/llvm-9.0.1.patchset

```diff
@@ -120,5 +120,27 @@ index 634c16a..4ab46b9 100644
  #ifdef HAVE_BACKTRACE
  # include BACKTRACE_HEADER         // For backtrace().
 -- 
-2.21.0
+2.37.3
+
+
+From afa6965c3ee577352fa2d6ef4518f8bbdb6a40d0 Mon Sep 17 00:00:00 2001
+From: Begasus <begasus@gmail.com>
+Date: Mon, 8 May 2023 20:51:14 +0200
+Subject: fix for: error: 'numeric_limits' is not a member of 'std'
+
+
+diff --git a/utils/benchmark/src/benchmark_register.h b/utils/benchmark/src/benchmark_register.h
+index 0705e21..c731ca5 100644
+--- a/utils/benchmark/src/benchmark_register.h
++++ b/utils/benchmark/src/benchmark_register.h
+@@ -4,6 +4,7 @@
+ #include <vector>
+ 
+ #include "check.h"
++#include <limits>
```

Comment:
I would sort this further up, together with `<vector>` (they both are standard library headers).

Reply:
Fixed :)